youtube_dl/extractor/libraryofcongress.py

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+class LibraryOfCongressIE(InfoExtractor):
+    IE_NAME = 'LibraryOfCongress'
+    _VALID_URL = r'https?://www.loc.gov/item/(?P<id>[0-9]+)'
+
+    _TESTS = [{
+        'url': 'http://loc.gov/item/90716351/',
+        'info_dict': {
+            'id': '90716351',
+            'ext': 'mp4',
+            'title': 'Pa\'s trip to Mars /'
+        },
+        'params': {
+            # m3u8 download
+            'skip_download': True,
+        }
+    }, {
+        'url': 'https://www.loc.gov/item/97516576/'
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, video_id)
+
+        self.report_extraction(video_id)
+        
+        m3u8_id = re.search(r'media_service [0-9]{7}-[0-9]-[0-9]', webpage).group()[14:]
+        m3u8_url = 'http://stream-media.loc.gov/hls-vod/media/ls/sagan/%s.mp4.m3u8' % m3u8_id
```

Comment:
Correct approach is to mimic web browser:
1. Use https://media.loc.gov/services/v1/media?callback=jQuery18205494330186516222_1463503121265&id=E6AB0B2585930180E0438C93F0280180&context=jsonp&_=1463503124082
2. Mimic web player url construction:
```js
        // generate hls live streaming url over https
        url = url.replace("rtmp", "http");
        
        // some urls come in with no extension and we need it on there for access
        if(url.indexOf('.mp4') == -1 && url.indexOf('.mp3') == -1){
            url += (is_video ? '.mp4' : '.mp3');
        }
        
        if(url.indexOf('vod/mp4:') > -1 ){
            url = url.replace("vod/mp4:", "hls-vod/media/")+".m3u8";
        } else if ( url.indexOf('vod/mp3:') > -1 ){
            url = url.replace("vod/mp3:", "");
        }
```